src/m_menu.c

```diff
@@ -3916,6 +3917,24 @@ static void M_SetMidiPlayer(void)
   S_RestartMusic();
 }
 
+static void M_EnableDisableFPSLimit(void)
+{
+  if (!uncapped)
+  {
+    gen_settings1[gen1_fpslimit].m_flags |= S_DISABLE;
+  }
+  else
+  {
+    gen_settings1[gen1_fpslimit].m_flags &= ~S_DISABLE;
+  }
+}
+
+static void M_CoerceFPSLimit(void)
+{
+  if (fpslimit > 0 && fpslimit < FPS_LIMIT_MIN)
```

Comment:
Why check for `> 0`? What if the value is e.g. `-1`? Granted, that's not possible due to the config system, but this restriction is strictly not necessary.

Reply:
Yep, you are absolutely correct. This is vestigial code from when I copied over some of the Crispy implementation.